spec/spec.md

```diff
@@ -1069,7 +1069,9 @@ database. There may be more or fewer columns.
        "Hib3",   1996,    50,     "AGO",       "Angola",          5000,     1000,      NA,    5670
        "Hib3",   1997,    50,     "AGO",       "Angola",          6000,     1200,      NA,    5870
 
+
 ## POST /modelling-groups/{modelling-group-id}/responsibilities/{touchstone-id}/{scenario-id}/estimates/
+** DEPRECATED **
```

Comment:
Rather than modify an endpoint that is currently in use, we should add new endpoints, so that the API is backwards compatible and can be deployed before the portals have been updated.